### DISCUSSION TITLE
Create a branch for working on an issue

### DISCUSSION THREAD

**Discussion by ali5h** [magit/forge#343] [Ideas]
is there an easy way to create an empty commit that has the body of the issue as its commit msg?

**Comment by tarsius** [comment]
There's no command for that, so doing this is a multi-step process and I think that's okay.

**Reply by ali5h** [reply]
so what is the shortest path?

**Reply by tarsius** [reply]
Please describe in more detail what you want to do. I.e. instead of asking me to provide you with the "shortest path" based on a vague description of what you want to achieve, first describe the path that you consider to be too long.

**Reply by ali5h** [reply]
for example, to work on an issue one must create a branch like `USER-issue-#issuenum` and then create an empty commit based on the content and title of the issue, so one can amend to it

**Reply by tarsius** [reply]
> so what is the shortest path?

It's what you are [almost certainly] already doing:

1. Create the branch using `b c USER-issue-#issuenum RET`.
2. Visit the issue by placing the cursor on it and pressing `RET`.
3. Kill the title and content.
4. Initiate a commit with `c - e c`.
5. Yank the title and content. Markup the former as such.
6. Finish the commit.

Of course that could be automated but I don't intend to implement that.

**Reply by tarsius** [reply]
I still don't plan to do that and since I haven't heard back in a few years now, I am also closing now.
